docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md

```diff
@@ -0,0 +1,256 @@
+   running: true
+   template:
+       metadata:
+       labels:
+           kubevirt.io/vm: vmo-jh
+       spec:
+           domain:
+               cpu:
+                   cores: 2
+                   sockets: 2
+                   threads: 1
+               devices:
+                   disks:
+                   - disk:
+                       bus: virtio
+                       name: datavolume-os
+                   - disk:
+                       bus: virtio
+                       name: cloudinitdisk
+                   interfaces:
+                   - masquerade: {}
+                       model: virtio
+                       name: default
+               machine:
+                   type: q35
+               resources:
+                   requests:
+                       memory: 8Gi
+           networks:
+           - name: default
+               pod: {}
+           volumes:
+           - dataVolume:
+               name: vmo-jh-dv
+               name: datavolume-os
+           - cloudInitNoCloud:
+               userData: |
+                   packages:
+                   - qemu-guest-agent
+                   runcmd:
+                   - ["sudo", "systemctl", "enable", "--now", "qemu-guest-agent"]
+               name: cloudinitdisk
+   EOF
+   ```
+
+   :::warning
+
+   Depending on the configuration of your VMO cluster's CSI layer, you may need to change the configured `ReadWriteMany`
+   access mode to `ReadWriteOnce`. Check the documentation for your chosen CSI to ensure you have the correct setting.
+
+   :::
+
+4. The VM provisioning and start up process will take a few minutes to complete. Execute the following command to check
+   the status of your VMs.
+
+   ```shell
+   kubectl get vms --all-namespaces
+   ```
+
+   The `vmo-jh` VM appears in the state `Running` when it is ready for usage.
+
+   ```shell hideClipboard
+   NAMESPACE              NAME                                 AGE     STATUS          READY
+   default                vmo-jh                               6m16s   Running         True
+   ```
+
+5. The VM is created with the `Welcome@123` default password and is accessible through your cluster's kubeconfig file.
+   You can secure access to the cluster by uploading a public key to the VM and adding it as an authorized key. You will
+   then need to supply the corresponding private key when connecting to the VM.
+
+   ```shell
+   virtctl scp public-key.pub root@vmo-jh.default:/root
+
+   virtctl ssh root@vmo-jh.default --local-ssh=true
+
+   mkdir -p /root/.ssh && cat /root/public-key.pub >> /root/.ssh/authorized_keys
+   ```
+
+6. Copy your cluster's kubeconfig file to the VM. Replace the placeholders with local path to the kubeconfig file and
+   the path to your private key. The kubeconfig file is required for subsequent operations.
+
+   ```shell
+   virtctl scp <path-to-downloaded-kubeconfig-file> -i <path-to-private-key> root@vmo-jh.default:/root
+   ```
+
```

Comment:
Please insert a step here explaining that you must download the latest palette CLI release binary to the jump host.

Reply:
At first I was, then I decided that I don't want to have to rebuild the jump host image each time we release the CLI.

Reply:
Added this step now.